cmd/copy.go

```diff
@@ -0,0 +1,111 @@
+//
+// DISCLAIMER
+//
+// Copyright 2020 ArangoDB GmbH, Cologne, Germany
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Copyright holder is ArangoDB GmbH, Cologne, Germany
+//
+// Author Gergely Brautigam
+//
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	"github.com/arangodb-managed/arangocopy/pkg"
+)
+
+func init() {
+	InitCommand(
+		RootCmd,
+		&cobra.Command{
```

Comment:
IMO copy should be the root command. `arangocopy copy ...` looks strange

Reply:
So you'd prefer the binary to be called `copy`? 

Right now this is the way of calling this:

```
./bin/darwin/amd64/arangocopy copy --source-address tcp://localhost:8529 --destination-address https://094c6fa1709c.arangodb.cloud:8529/ --destination-username root --destination-password <> --batch-size 10000
```

Reply:
```
./bin/darwin/amd64/arangocopy help
ArangoDB Copy. The copy tool of the ages.

Usage:
  arangocopy [flags]
  arangocopy [command]

Available Commands:
  copy        Copy data from source to destination.
  help        Help about any command
  version     Show the current version of this tool

Flags:
  -h, --help   help for arangocopy

Use "arangocopy [command] --help" for more information about a command.
```

Reply:
`The copy tool of the ages.` -> placeholder. I'll have something sensible here. :)

Reply:
`arangocopy --source-address .... --destination-addres...`

Reply:
Oh gotcha!